manifests/m/Microsoft/Edge/Canary/112.0.1705.0/Microsoft.Edge.Canary.installer.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Microsoft.Edge.Canary
+PackageVersion: 112.0.1705.0
+Installers:
+- Architecture: x64
+  InstallerType: exe

```

Comment:
All of the other Microsoft.Edge distributions are Wix distributions, so this being different seems a bit off to me

Reply:
No idea how to identify Wix but this installer properties seem different from others
![image](https://user-images.githubusercontent.com/9417531/221750735-4d7c7841-722b-46fb-a6d6-989de8188d5f.png)